src/features/theme-switcher/model.ts

```diff
@@ -0,0 +1,8 @@
+import { createEvent, createStore } from "effector";
+
+const defaultDark = window.matchMedia("(prefers-color-scheme: dark)").matches;
+export const $theme = createStore(defaultDark? "dark" : "light");
```

Comment:
nit
in my opition we should use enums here
sorry, best way will be use enums
```
export enum Themes {
    DEFAULT = 'default',
    DARK = 'dark'
}
```

Reply:
done